pkg/utils/types.go

```diff
@@ -348,3 +349,16 @@ func NewClusterRoleBinding(bindingName, roleName string, subjects []rbac.Subject
 		Subjects: subjects,
 	}
 }
+
+func ServiceMonitor(serviceMonitorName, namespace string) *monitoringv1.ServiceMonitor {
+	return &monitoringv1.ServiceMonitor{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       monitoringv1.ServiceMonitorsKind,
+			APIVersion: monitoringv1.Group + "/" + monitoringv1.Version,
```

Comment:
Can this not be similar to `APIVersion: batch.SchemeGroupVersion.String()` for SM?

Reply:
I could not find SchemeGroupVersion for "github.com/coreos/prometheus-operator/pkg/client/monitoring/v1".   It is available for "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1", though.  But I could not use it in monitoring.go...